### PR TITLE
Custom folder with JAVA files

### LINKED ISSUE #252: ML pipeline; Custom folder of JAVA files
1. Able to use for training custom folder 
2. custom folder defined using  environment variable "JAVA_FILES_PATH"
3. If JAVA_FILES_PATH is not defined use old behavior

### PR DESCRIPTION
**Pull request by acheshkov**
Now we can set up custom folder with JAVA files to train model. closes #252

### CODE REVIEW

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/253#issuecomment-620857533))

@acheshkov Job `gh:yegor256/aibolit#253` is not assigned, can't get performer

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/253#issuecomment-620857533))

@acheshkov Pull request [#253](https://github.com/yegor256/aibolit/issues/253) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

File: aibolit/ml_pipeline/ml_pipeline.py [hunk 1]
```diff
@@ -11,16 +11,25 @@
 from aibolit.config import CONFIG
 
 
+JAVA_FILES_PATH = os.environ['JAVA_FILES_PATH']
+
 def collect_dataset():
```
Comment (lyriccoder) [inline aibolit/ml_pipeline/ml_pipeline.py:16]: collect _dataset now has a parameter with java folder, no need to use os.environ. Fix it

File: aibolit/ml_pipeline/ml_pipeline.py [hunk 2]
```diff
@@ -11,16 +11,25 @@
...
     """
     Run bash scripts to collect metrics and patterns for java files
```
Comment (lyriccoder) [inline aibolit/ml_pipeline/ml_pipeline.py:18]: Fix all confilicts

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#253` is not assigned, can't get performer

**Pull request closed without merging**